tasks.py

```diff
@@ -32,7 +32,7 @@ def analyze(ctx):
 @task
 def test(ctx):
     """Run unit tests."""
-    ctx.run('pytest --cov pytest_voluptuous --cov-report term-missing')
+    ctx.run('coverage run --append -m pytest')
```

Comment:
We lose the coverage report (by `pytest-cov`) after each run now. Can you add `coverage report --show-missing` as a second step?